### PR TITLE
Fix show-more toggle for datafiles

### PR DESCRIPTION
**Pull request by TobyRet**

### CODE REVIEW

**Comment by laurentqro** [comment]
Works well. Only small niggle: mouse pointer (finger) does not appear when I hover the "show more/less" link.

File: app/assets/stylesheets/components.scss [hunk 1]
```diff
@@ -225,6 +226,11 @@
   &__download {
     margin-top: 1.5em;
   }
+  .show-toggle {
```
Comment (maxf) [inline app/assets/stylesheets/components.scss]: This should use the BEM convention now.

File: app/views/datasets/_datafile_table.html.erb [hunk 2]
```diff
@@ -6,39 +6,45 @@
...
-          <%= link_to (datafile.name ? datafile.name : 'Data Link'),
-              datafile.url,
-              :data => {:'ga-event' => "datafile-download"}
-          %>
-        </td>
-        <% if datafile.format.blank? %>
-          <td class="dgu-unavailable">N/A</td>
-        <% else %>
-          <td><%= datafile.format.upcase %>
+  <% datafiles.each do |datafile| %>
```
Comment (maxf) [inline app/views/datasets/_datafile_table.html.erb]: can you indent under `<tbody>` ?

File: app/views/datasets/_datafile_table.html.erb [hunk 3]
```diff
@@ -6,39 +6,45 @@
...
-        </tr>
-      <% end %>
+      </td>
+    </tr>
+  <% end %>
   </tbody>
 </table>
+
+<% if datafiles.length > 5 %>
+  <p class="show-toggle">
```
Comment (maxf) [inline app/views/datasets/_datafile_table.html.erb]: this shoud be a button, for accessibility reason. Hence you can reuse the button.secondary class from govuk-overrides.scss

File: app/assets/stylesheets/components.scss (hunk 1 above)
Comment (TobyRet) [reply app/assets/stylesheets/components.scss]: This is overriding a gov uk class. 
Have changed to 'dgu-datalinks__toggle'

File: app/views/datasets/_datafile_table.html.erb (hunk 3 above)
Comment (TobyRet) [reply app/views/datasets/_datafile_table.html.erb]: Cool - done

File: app/assets/stylesheets/components.scss (hunk 1 above)
Comment (TobyRet) [reply app/assets/stylesheets/components.scss]: No longer required as we are now using 'button secondary' as requested below

**Comment by TobyRet** [comment]
@laurentqro - hey mate thanks for the comment - point has been fixed

**Comment by laurentqro** [comment]
🙌🏻

File: app/views/datasets/_datafile_table.html.erb [hunk 4]
```diff
@@ -7,38 +7,42 @@
...
-          <%= link_to (datafile.name ? datafile.name : 'Data Link'),
-              datafile.url,
-              :data => {:'ga-event' => "datafile-download"}
-          %>
-        </td>
-        <% if datafile.format.blank? %>
-          <td class="dgu-unavailable">N/A</td>
-        <% else %>
-          <td><%= datafile.format.upcase %>
+    <tr class="<%= show_more?(datafiles, datafile) %> dgu-datafile">
```
Comment (maxf) [inline app/views/datasets/_datafile_table.html.erb]: Identation was correct there. Maybe I wasn't clear, sorry: so far we've indented the html inside the <% for/if %> statements (like below). I'm happy to discuss changing that, but we should be consistent

**Pull request merged**